index.html

```diff
@@ -18,7 +18,67 @@
         rel="stylesheet">
 </head>
 
-<body><!--mobile menu-->
+<body>
+    <!--pop up window-->
+        <article id="popup" class="popup">
+            <div class="popupHeader">
+                <h2>Tonic</h2>
+                <div id="closePopUp">&times;</div>
+            </div>
+
+
+
+          <div class="popupAfterHeader">
+            <div class="canopy">
+                    <h4 class="colorh4 margin10 padtop">CANOPY</h4>
+                <ul class="popupAfterHeader">
+                    <li class="canopylist popupFlex margin10">
+                        <span>
+                            <img width="8px" height="8px" src="images/Ellipse 12.png" alt="">
+                        </span>
+                        <p>Back End Dev</p>
+                    </li>
+                    <li class="canopylist popupFlex margin10">
+                        <span>
+                            <img width="8px" height="8px" src="images/Ellipse 12.png" alt="">
+                        </span>
+                        <p>2015</p>
+                    </li>
+                </ul>
+            </div>
+        </div>
+            <div class="scrnshot width">
+                <img class="mobile" src="images/Snapshoot Portfolio (2).png" alt="">
+                <img class="desktop" src="images/Snapshoot Portfolio - Copy.png" alt="">
+            </div>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus amet et vero architecto, dolor modi eligendi animi
+                praesentium, eum eos cumque reprehenderit dolorem voluptates optio provident nemo asperiores libero ratione.
+            </p>
+            <ul class="popupFlex">
+                <li class="languages">HTML</li>
+                <li class="languages">CSS</li>
+                <li class="languages">JavaScript</li>
+            </ul>
+            <div class="popupFlex">
+                <div class="seeproject margin10">
+                    <a title="#" href="#">see live</a>
+                </div>
+                <div class="seeproject">
+                    <a title="#" href="#">see source</a>
+                </div>
+            </div>
+        </article>
+    <div id = 'overlay' class="overlay"></div>
```

Comment:
- [ ] For all your projects, the popup window shows the same project details. The requirement is that the popup should show project details specific to a particular project. I suggest you used javascript & dynamically display the project details.  😄 

- [ ] Your popup window on the desktop version appears on the left. Please position it in the center as is required in the design. 

- [ ] Your mobile window popup is not displayed in Figma. You don't have the padding in the design. Please make sure the mobile popup matches the design. Good-looking
**Figma**
![mobile](https://github.com/MPDADDY/portfolio/assets/99656687/4f57aefa-90db-44de-96cb-6d4f1a762d30)
 projects increase your chances of getting a job.
 
 **Your design**
 
![mobile2](https://github.com/MPDADDY/portfolio/assets/99656687/7062bbca-45ad-4823-b718-62c59a2cbe64)